conf/application.conf

```diff
@@ -134,6 +134,13 @@ microservice {
       port = 9303
       #This is the port for the income-tax-submission-stub
     }
+
+    if {
+        host = "localhost"
+        port = 9084
+        authorization-token = "localIFToken"
```

Comment:
token should be retrievable from AppConfig.scala file. 

We've laid it out like so in other places as the IF token will be different for different APIs

`authorisation-token {
        1647 = "secret"
        1645 = "secret"
        1661 = "secret"
        1664 = "secret"
        1662 = "secret"
        1643 = "secret"
}`

Could do something similar if you expect there will be multiple IF APIs https://github.com/hmrc/income-tax-employment/blob/main/conf/application.conf#L152